src/databricks/labs/ucx/source_code/base.py

```diff
@@ -167,7 +167,12 @@ class Fixer(ABC):
 
     @property
     @abstractmethod
-    def name(self) -> str: ...
+    def diagnostic_code(self) -> str | None:
```

Comment:
This rename made the intended purpose more clear, see `LinterContext.fixer`, the only place it was used